### PR TITLE
Added Logo SOI + Text Update

### PR DESCRIPTION
**Pull request by marcmerce**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,5 +26,8 @@
     "gulp-concat": "^2.6.0",
     "gulp-sass": "^2.3.2",
     "gulp-uglify": "^2.0.0"
+  },
+  "dependencies": {
+    "gulp-install": "^1.1.0"
```
Comment (EFF) [inline package.json]: what the heck is this

Comment (EFF) [reply package.json]: If this is necessary, please ad this as a dev dependency

**Review by EFF — APPROVED** [review]
small fix then LGTM

**Review by dalloriam — APPROVED** [review]
LGTM

**Pull request merged**